en/flight_controller/pixhawk_mini.md

```diff
@@ -170,6 +171,13 @@ ST Microelectronics® and a NuttX real-time operating system, delivering
 excellent performance, flexibility, and reliability for controlling any
 autonomous vehicle.
 
+## Known issues
+
+- The Pixhawk mini has a [hardware defect](https://github.com/PX4/Firmware/issues/7327#issuecomment-317132917) that makes the internal MPU9250 IMU unreliable.

```

Comment:
The issue is only affecting earlier revisions of the board that released by the manufacturer but has since corrected the issue on a later revision.

https://github.com/PX4/Firmware/issues/7327#issuecomment-372393609
> Hi this is a hardware issue there's a pin that was not routed it was left open and is causing a problem with the mpu9250. I believe this was fixed on later revisions of the board

Reply:
Yes. We should also say what you do if you have the bad version, and how to detect the bad version.